coding/multilang_utf8_string.cpp

```diff
@@ -56,14 +56,14 @@ int8_t StringUtf8Multilang::GetLangIndex(string const & lang)
 // static
 char const * StringUtf8Multilang::GetLangByCode(int8_t langCode)
 {
-  if (langCode < 0 || langCode > g_languages.size() - 1)
+  if (langCode < 0 || langCode >= static_cast<int8_t>(g_languages.size()))
```

Comment:
I'm suggesting you to do following:
```
if (langCode < 0 || static_cast<size_t>(langCode) >= g_languages.size())
  ...
```
becase in the second term of disjunction it's guaranteed that langCode is non-negative due to lazy evaluation of conjunctions/disjunctions. So, langCode is non-negative, g_languages is non-negative, let's select the wider type (size_t in this case) and cast langCode.